config/config.yml

```diff
@@ -183,6 +183,7 @@ arisa:
         - launcher_accounts.json
         - launcher_msa_credentials.json
         - launcher_profiles.json
+        - .jfr
```

Comment:
Does this actually work? If I see that correctly `PrivacyModule` checks whether the file names match exactly.

Reply:
I actually saw a contains

Reply:
https://github.com/mojira/arisa-kt/blob/master/src/main/kotlin/io/github/mojira/arisa/modules/PrivacyModule.kt#L55

Reply:
But that checks if any of the attachment names is contained in the set of `sensitiveFileNames`, isn't it?

We could probably change `sensitiveFileNames` to use regex patterns though, or add a separate `sensitiveFileExtensions`, but that might clutter the class a bit.